osuosl/streamwebs
# Feat/gallery

Testing this PR

- Navigate to a site with existing gallery data, such as Foster Salmon Watch
- Notice that the page contains a list of datasheets, as well as a list of gallery items, organized by school
- The bottom of the page should have three buttons, one of which is "Add Gallery Item"
- Each type of gallery item has an add screen which adds a gallery item to the site
- Click an existing gallery item and see the view page for an album, image, or file
- If you are logged in as an Editor+, you should see a delete button for the gallery item
- The data import scripts should import drupal site data correctly - no site should have gallery items (images, albums, files) with missing data
